account_payment_netting/views/account_invoice_view.xml

```diff
@@ -0,0 +1,38 @@
+<!-- Copyright 2019 Ecosoft
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+      <record id="action_invoice_all_tree" model="ir.actions.act_window">
+          <field name="name">Invoices to Netting</field>
+          <field name="res_model">account.invoice</field>
+          <field name="view_type">form</field>
+          <field name="view_mode">tree,form</field>
+          <field eval="False" name="view_id"/>
+          <field name="domain">[('state', '=', 'open')]</field>
+          <field name="context">{'type':'out_invoice', 'journal_type': 'sale'}</field>
+          <field name="search_view_id" ref="account.view_account_invoice_filter"/>
+          <field name="help" type="html">
+            <p class="o_view_nocontent_smiling_face">
+              Create a customer invoice
+            </p><p>
+              Create invoices, register payments and keep track of the discussions with your customers.
+            </p>
+          </field>
+      </record>
+
+      <record id="action_invoice_all_view1" model="ir.actions.act_window.view">
+          <field eval="1" name="sequence"/>
+          <field name="view_mode">tree</field>
+          <field name="view_id" ref="account.invoice_tree_with_onboarding"/>
+          <field name="act_window_id" ref="action_invoice_all_tree"/>
+      </record>
+
+      <record id="action_invoice_all_view2" model="ir.actions.act_window.view">
+          <field eval="2" name="sequence"/>
+          <field name="view_mode">form</field>
+          <field name="view_id" ref="account.invoice_form"/>
+          <field name="act_window_id" ref="action_invoice_all_tree"/>
+      </record>
+
+      <menuitem action="action_invoice_all_tree" id="menu_action_invoice_all_tree" parent="account.menu_finance_entries" sequence="50"/>
```

Comment:
I'd personally put the menu items next to "Customers > Payments" and "Vendors > Payments" (I suppose having the same view in two separate places is OK for this case).

One concern of having the menu item under `account.menu_finance_entries` is that some users may be confused for not enabling "Show Full Accounting Features".

Reply:
Thank you @yostashiro I think oca-gen-addon-readme is not mandatory, as long as we have readme folder it will be auto generated once it is merged (I am yet to see too).
Yes, we intended for only full accounting user to use it at the moment, I think it is not much generic feature (we only have 1 customer request to use it).